fdbserver/fdbserver.actor.cpp

```diff
@@ -1680,6 +1680,13 @@ int main(int argc, char* argv[]) {
 		const auto opts = CLIOptions::parseArgs(argc, argv);
 		const auto role = opts.role;
 
+#ifdef _WIN32
+		//For now, ignore all tests for Windows
+		if (role == ServerRole::Simulation || role == ServerRole::UnitTests || role == ServerRole::Test) {
```

Comment:
Perhaps output something like "FDB does not support tests in Windows at this stage", so people are not surprised in that case?

Reply:
Output added